packages/x-components/src/x-modules/history-queries/components/history-query.vue

```diff
@@ -91,8 +91,32 @@
 
 This component only requires a prop called `suggestion`
 
-```vue
-<HistoryQuery :suggestion="historyQuery" />
+```vue live
+<template>
```

Comment:
`<template>` can only have one root child. Wrap the components in a div for the code to work